konduit-serving-python/src/main/java/ai/konduit/serving/executioner/PythonExecutioner.java

```diff
@@ -607,13 +607,13 @@ public static void execWithSetupAndRun(String code) {
         acquireGIL();
         _exec(code);
         if (_hasGlobalVariable("setup") && _hasGlobalVariable("run")){
-            log.debug("setup() and run() methods found.");
+            log.info("setup() and run() methods found.");
```

Comment:
Mind adding comments around context in the various run modes we're doing here?

Reply:
added in #92